Formula/tezos-sapling-params.rb

```diff
@@ -13,8 +13,6 @@ class TezosSaplingParams < Formula
 
   bottle do
     root_url "https://github.com/serokell/tezos-packaging/releases/download/#{TezosSaplingParams.version}/"
-    sha256 cellar: :any, mojave: "4e89932b0626cffe80214ba45342280c340b34c58ebbf7c3e0185a6d4662732d"
-    sha256 cellar: :any, catalina: "5f7a5687d67051eafcfb7cb5ac542143a325a135403daeca6595602bfd400441"
```

Comment:
Is this necessary? Considering the Formula and bottles won't change?

Reply:
Ah, you're right :sweat_smile: The lesson in clearly not to use `sed` but to go through the files painstakingly by hand :fine: